cudl-sandbox/terraform.tfvars

```diff
@@ -37,10 +34,23 @@ source-bucket-sns-notifications = [
         "raw"        = true
       },
     ]
+  },
+  {
+    "bucket_name"   = "cudl-data-releases"
+    "filter_prefix" = "collections/",
+    "filter_suffix" = ".json"
+    "subscriptions" = [
+      {
+        "queue_name" = "CUDLIndexCollectionQueue",
+        "raw"        = true
+      },
+      {
+        "queue_name" = "CUDLPackageDataUpdateDBQueue",
+        "raw"        = true
+      },
+    ]
   }
 ]
-// NOTE: If you are adding anything here you need to add a code block to
-// the s3.tf file
 source-bucket-sqs-notifications = [
```

Comment:
Would it make sense to rename this variable to `transform-lambda-bucket-sqs-notifications` in the `cudl-data-processing` module as the existing name is confusing given the list can now also contain the dest bucket?

Reply:
Renamed.